README.md

```diff
@@ -159,6 +159,207 @@ python ./scripts/train_hf_ner.py ./data/annotations/train.spacy ./data/annotatio
+```
+
+This will create a `predictions.jsonl` file that can be loaded with the
+[`textcat.manual`](https://prodi.gy/docs/recipes#textcat-manual) recipe.
+
+Note that the OpenAI API might return "429 Too Many Request" errors when
+requesting too much data at once - in this case it's best to ensure you only
+request 100 or so examples at a time and take a look at the [API's rate
+limits](https://help.openai.com/en/articles/5955598-is-api-usage-subject-to-any-rate-limits).
+
+### Working with imbalanced data
+
+The `textcat.openai.fetch` recipe is suitable for working with datasets where
+there is severe class imbalance. Usually, you'd want to find examples of the
+rare class rather than annotating a random sample. From there, you want to
+upsample them to train a decent model and so on.
+
+As it turns out, **large language models (LLMs) are effective at finding the
+rare class.** Using the [Reddit r/cooking dataset](data), we prompted ChatGPT to
+look for comments that resemble a food recipe. Instead of annotating 10,000
+examples, we ran `textcat.openai.fetch` and obtained 145 positive classes. Out
+of those 145 examples, 114 turned out to be true positives (79% precision). We
+then checked 1,000 negative examples and found 12 false negative cases (98%
+recall). 
+
+Ideally, once we fully annotated the dataset, we can train a supervised model
+that is better to use than relying on zero-shot predictions for production. The
+running cost is low and it's easier to manage.
+
+## Exporting the annotations and training a text categorization model
+
+After you've curated a set of predictions, you can export the results with
+[`db-out`](https://prodi.gy/docs/recipes#db-out):
+
+```bash
+python -m prodigy db-out my_textcat_data  > textcat_data.jsonl
+```
+
+The format of the exported annotations contains all the data you need to train a
+smaller model downstream. Each example in the dataset contains the original
+text, the tokens, span annotations denoting the entities, etc.
+
+You can also export the data to spaCy's [binary
+format](https://spacy.io/api/data-formats#training), using
+[`data-to-spacy`](https://prodi.gy/docs/recipes#data-to-spacy). This format lets
+you load in the annotations as spaCy `Doc` objects, which can be convenient for
+further conversion. The `data-to-spacy` command also makes it easy to train a
+text categorization model with spaCy. First you export the data, specifying the
+train data as 20% of the total:
+
+```bash
+# For binary textcat
+python -m prodigy data-to-spacy ./data/annotations/ --textcat my_textcat_data -es 0.2
+# For multilabel textcat
+python -m prodigy data-to-spacy ./data/annotations/ --textcat-multilabel my_textcat_data -es 0.2
+```
+Then you can train a model with spaCy:
+
+```bash
+python -m spacy train ./data/annotations/config.cfg --paths.train ./data/annotations/train.spacy --paths.dev ./data/annotations/dev.spacy -o textcat-model
+```
+
+This will save a model to the `textcat-model/` directory.
```

Comment:
Good section to add! One thing though; is there a reason why you didn't mention the `prodigy train` command?

Reply:
Ah good question! I followed the NER example here for consistency. I'll probably mention prodigy train as well.